src/Symfony/Component/Translation/Translator.php

```diff
@@ -351,15 +351,13 @@ protected function initializeCatalogue($locale)
     private function initializeCacheCatalogue($locale, $forceRefresh = false)
     {
         if (isset($this->catalogues[$locale])) {
-            return;
+            return; // don't see how this could ever happen...?
```

Comment:
you can delete it as it useless https://github.com/webfactory/symfony/blob/translator-tweaks/src/Symfony/Component/Translation/Translator.php#L353-L355